creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -220,6 +227,7 @@ router.post(
 )
 router.post(
   '/merge_primary_and_secondary',
+  ensurePrimaryMiddleware,
   handleResponse(mergePrimaryAndSecondaryController)
```

Comment:
lol so i looked at what the middleware internally does and pretty sure this will fail
https://github.com/AudiusProject/audius-protocol/blob/theo-orphaned-data-job/creator-node/src/middlewares.js#L103

it requires some request parameters that you don't pass in from the orphan queue
will let you decide on simplest path here but def worth re-testing stuff end to end with scope of changes here

Reply:
yep you're totally right: this gives a "User must be logged in" error. wdyt of removing the auth middleware? it requires a session, but I won't have an auth token to send since the user isn't taking any action here. with the guardrails I added I think this should be safe, but I could also add auth (maybe after testing on staging?) to be extra safe

Reply:
yeah agreed, waste of time - lets remove